src/platform_impl/macos/event.rs

```diff
@@ -241,8 +241,16 @@ pub fn check_function_keys(string: &str) -> Option<VirtualKeyCode> {
     None
 }
 
+const NS_LEFT_ALTERNATE_KEY_MASK: u64 = 0x000020 | NSEventModifierFlags::NSAlternateKeyMask.bits();
+const NS_RIGHT_ALTERNATE_KEY_MASK: u64 = 0x000040 | NSEventModifierFlags::NSAlternateKeyMask.bits();
```

Comment:
Do you have some documentation for these magic values? Whether you do or not, a bit of documentation on them would be nice!

Reply:
I couldn't find any official documentation from apple on how these key masks should be formulated but I found [this discussion which includes some useful formulas](https://community.folivora.ai/t/could-you-provide-a-formula-for-determining-keyboard-shortcuts-values/3606) and these key mask values are used in many projects such as SFML [permalink of the code snippet](https://github.com/fishbupt/sfml/blob/86694bcf5850045ff8bc9b6e2eb3874ee40cacec/src/SFML/Window/OSX/SFKeyboardModifiersHelper.mm#L43-L44) and [iTerm](https://github.com/gnachman/iTerm2/blob/14148ffa9510e13a52043e64b552d7bf6c44307c/sources/iTermKeyboardHandler.h#L15-L16).